sitback-furniture/src/components/cards/category-card/category-card.jsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from 'react-router-dom';
+import Button from '../../button/button';
+import styles from './category-card.module.scss'
+import { buttonData } from '../../../constants/string-constant';
+
+const CategoryCard = ({id,photo,category,description})=>{
+
+    return(
+        <div className={styles['card-wrapper']}>
+            <div className={styles['img-wrapper']}>
+                <img src={photo} alt={id} onerror="this.onerror=null;this.src='../../../assets/images/phillip-goldsberry-fZuleEfeA1Q-unsplash.jpg';"/>
```

Comment:
Could you create a re-usable image component?

Reply:
created re-usable image component